sdks/go/alpha.go

```diff
@@ -172,3 +173,37 @@ func (a *Alpha) SetCounterCapacity(key string, amount int64) (bool, error) {
 	}
 	return true, err
 }
+
+// Alpha().ListAppend(key, value): boolean
+// Alpha().ListDelete(key, value): boolean
+
+// GetListCapacity returns the Capacity for a List, given the List's key (name).
+// Will error if the key was not predefined in the GameServer resource on creation.
+func (a *Alpha) GetListCapacity(key string) (int64, error) {
+	list, err := a.client.GetList(context.Background(), &alpha.GetListRequest{Name: key})
+	if err != nil {
+		return -1, errors.Wrapf(err, "could not get List %s capacity", key)
+	}
+	return list.Capacity, nil
+}
+
+// SetListCapacity sets the capacity for a given list. Capacity must be between 1 and 1000.
+// TODO: Should capacity be between 0 and 1000?
```

Comment:
[Yep](https://github.com/googleforgames/agones/issues/2716#:~:text=Capacity%20can%20be%20no%20longer%20than%201000%20items.%20This%20could%20possibly%20be%20expanded%20in%20the%20future%20depending%20on%20use%20cases%20and/or%20performance.)!

> Capacity can be no longer than 1000 items. This could possibly be expanded in the future depending on use cases and/or performance.

Reply:
Ok, in the [#2716](https://github.com/googleforgames/agones/issues/2716) it mentioned 1:1000, so wanted to double check it's 0:1000.